app/views/curate/people/show.html.erb

```diff
@@ -74,6 +74,6 @@
 </div>
 
 <div class="page-link">
-  <% this_page = Curate.configuration.application_root_url + polymorphic_path(@person) %>
+  <% this_page = Curate.configuration.application_root_url + polymorphic_path(@person)[1..20] %>
```

Comment:
Okay looking again - chomp won't work, but this will still break if the pid changes length.`